modules/sonobiBidAdapter.js

```diff
@@ -407,4 +409,21 @@ function _iframeAllowed() {
   return userSync.canBidderRegisterSync('iframe', BIDDER_CODE);
 }
 
+function getBidFloor(bid) {
+  if (!isFn(bid.getFloor)) {
+    return (bid.params.floor) ? bid.params.floor : null;
+  }
+
+  let floor = bid.getFloor({
+    currency: 'USD',
+    mediaType: '*',
+    size: '*'
+  });
+
+  if (isPlainObject(floor) && !isNaN(floor.floor) && floor.currency === 'USD') {
+    return floor.floor;
+  }
+  return '';
```

Comment:
this always returns something, making the else in the wrapping function redundant i think?